py/tracks/latest.toml

```diff
@@ -13,6 +13,7 @@ fixtures = [
 queries = [
     "../specs/group_by.toml",
     "../specs/like.toml",
+    "../specs/regex.toml",
```

Comment:
@mfussenegger Without the analyzer the query:
`select * from userVisits where "UserAgent" ~ '(?i).*\bWindows\sNT\s6.0\b.*' limit 10000` always return 0 rows, I'm not sure why I'm not so much into the indexes :-(